test/expected/servers.out

```diff
@@ -1,22 +1,79 @@
-     }                                        +
+-- Loads the config -- only needed because we're not using actual PostgREST in the tests
+SELECT postgrest.pre_config();
+ pre_config 
+------------
+ 
+(1 row)
+
+-- Sets the config; would be called if we called callable_root() (in main)
+CALL set_server_from_configuration();
+-- shows the custom server url specified in the configuration
+select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'servers');
+                  jsonb_pretty                   
+-------------------------------------------------
+ [                                              +
+     {                                          +
+         "url": "https://127.0.0.2:3000/",      +
+         "description": "URL from configuration"+
+     }                                          +
+ ]
+(1 row)
+
+-- shows the overriding custom server url specified by the user
+INSERT INTO servers (url, description, priority) VALUES ('https://www.example.com/api/', 'Overriding URL', 30);
+select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'servers');
+                  jsonb_pretty                  
+------------------------------------------------
+ [                                             +
+     {                                         +
+         "url": "https://www.example.com/api/",+
+         "description": "Overriding URL"       +
+     }                                         +
+ ]
+(1 row)
+
+-- shows the additional custom server url specified by the user
+INSERT INTO servers (url, description, priority, slug, variables)
+	VALUES ('https://www.example.com/otherapi/?s={s}&browser={browser}', 'Additional URL', 30, 'additional', '{"s": "worms", "browser": "none"}');
+select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'servers');
+                                jsonb_pretty                                 
+-----------------------------------------------------------------------------
+ [                                                                          +
+     {                                                                      +
+         "url": "https://www.example.com/api/",                             +
+         "description": "Overriding URL"                                    +
+     },                                                                     +
+     {                                                                      +
+         "url": "https://www.example.com/otherapi/?s={s}&browser={browser}",+
+         "variables": {                                                     +
+             "s": "worms",                                                  +
+             "browser": "none"                                              +
+         },                                                                 +
+         "description": "Additional URL"                                    +
+     }                                                                      +
```

Comment:
The error is here. The order of the json objects varies between pg versions. An explicit ORDER BY seems like the solution.